scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -233,7 +232,7 @@ def run_indv(self, suffix="base", st_archive=False):
 
         logger.info(infostr)
 
-        self._case.case_run(skip_pnl=self._skip_pnl, submit_resubmits=is_batch)
+        self._case.case_run(skip_pnl=self._skip_pnl, submit_resubmits=True)
```

Comment:
We're going to submit_resubmits even on non-batch machines?

Reply:
fixed